kong/plugins/influxdb-log/schema.lua

```diff
@@ -0,0 +1,9 @@
+local utils = require "kong.tools.utils"
+local http_log_schema = require "kong.plugins.http-log.schema"
+
+local influxdb_log_schema = utils.deep_copy(http_log_schema)
+
+influxdb_log_schema.fields.method = { default = "POST", enum = { "POST" } }
```

Comment:
@Tieske I also want to make it can not be configured and don't know what should I do, hope to get your help